test/test_bundle.js

```diff
@@ -841,6 +1035,53 @@ describe('json-pollock tests', function () {
 
   });
 
+  describe('render carousel select', function () {
+    const container = addToBody(JsonPollock.render(JSON.stringify(carouselSelectConf)));
+    const carouselRoot = container.children[0];
+    const carouselRootWrapper = container.children[0].children[0];
+    const carouselRootLayout = container.children[0].children[0];
+    const carouselListRoot =  carouselRootLayout.children[0];
+    const card1 = carouselRootLayout.children[0].children[0];
+    const card2 = carouselRootLayout.children[0].children[1];
+    const card3 = carouselRootLayout.children[0].children[2];
+
+    it('carousel root exist', function () {
+      chai.expect(carouselRoot.className).to.contain('lp-json-pollock');
+    });
+
+    it('carousel wrapper root exist', function () {
+      chai.expect(carouselRootWrapper.className).to.contain('lp-json-pollock-layout-carousel-wrapper');
+    });
+
+    it('carousel root layout exist', function () {
+      chai.expect(carouselRootLayout.className).to.contain('lp-json-pollock-layout-carousel');
+    });
+
+    it('carousel has aria-label', function(){
+      chai.expect(carouselListRoot.getAttribute('aria-label')).to.be.equal('Carousel with buttons');
+    });
+
+    it('carousel elements length equal to conf element length', function () {
+      chai.expect(carouselRootLayout.children.length).to.be.equal(carouselConf.elements.length);
+    });
+
+    it('carousel elements are in the right order', function () {
+      chai.expect(card1.children[0].innerText).to.be.equal('1');
+      chai.expect(card2.children[0].innerText).to.be.equal('2');
+      chai.expect(card3.children[0].innerText).to.be.equal('3');
+    });
+
+    it('carousel accessibility attrbs', function () {
+      chai.expect(carouselListRoot.getAttribute('role')).to.be.equal('list');
+      chai.expect(card1.getAttribute('role')).to.be.equal('listitem');
+      chai.expect(card2.getAttribute('role')).to.be.equal('listitem');
+      chai.expect(card3.getAttribute('role')).to.be.equal('listitem');
+    });
+    it('carousel root should have appropriate WCAG attribute', function () {
+      chai.expect(carouselRootWrapper.getAttribute('aria-label')).to.be.equal('Carousel');
+    });
```

Comment:
@dvakulenko Hey, it seems like this one also will always fail. There is no logic to add this aria-label in the code.